deployments/vanilla/terraform/main.tf

```diff
@@ -62,13 +80,15 @@ module "eks_blueprints" {
   vpc_id             = module.vpc.vpc_id
   private_subnet_ids = module.vpc.private_subnets
 
+  # configuration settings: https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/modules/aws-eks-managed-node-groups/locals.tf
   managed_node_groups = {
     mg_5 = {
       node_group_name = "managed-ondemand"
-      instance_types  = ["m5.large"]
+      instance_types  = [var.node_instance_type]
```

Comment:
There is a sense to have a separate node group for GPU instances. It is extremely pricy to run all kubeflow workloads on GPU instances.

Reply:
added